students/fortucj/lesson02/Lesson02_exercise2_Fizz_Buzz.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+print("""Fizz Buzz Exercise.
+      """)
+
+for i in range(1,101):
+    if (i%3==0) and (i%5!=0):
```

Comment:
You should have spaces around the operators: if (i % 3 == 0) etc.  For all in this file.